bayeux.go

```diff
@@ -43,9 +43,19 @@ type TriggerEvent struct {
 
 // Status is the state of success and subscribed channels
 type Status struct {
-	connected bool
-	clientID  string
-	channels  []string
+	connected    bool
+	clientID     string
+	channels     []string
+	connectCount int
+}
+
+func (st *Status) connect() {
+	st.connectCount++
+}
+
+func (st *Status) disconnect(out chan MaybeMsg) {
+	close(out)
```

Comment:
That's an anti-pattern in general, you shouldn't close channels you don't own/create.

Reply:
Sure, I will update it.